packages/core/src/fhir-to-cda/fhir-to-cda.ts

```diff
@@ -0,0 +1,8 @@
+import { Bundle } from "@medplum/fhirtypes";
+import { splitBundleByCompositions } from "./composition-splitter";
+import { generateCdaFromFhirBundle } from "./cda-generators";
+
+export function convertFhirBundleToCda(fhirBundle: Bundle): string[] {
```

Comment:
This is a packed function, let's add a TSDoc explaining what it does and what it returns, please? Like, what the array of string represent, etc.